src/component/rent/RentModalBook.tsx

```diff
@@ -32,14 +32,14 @@ const RentModalBook = ({
     const isAlreadySelected = selectedBooks.some(
       book => book.bookId === +bookId,
     );
-    if (isAlreadySelected)
+    if (isAlreadySelected) {
       addDialogWithTitleAndMessage(
         "alreadySelected",
         "이미 선택된 도서입니다.",
         "다시 한번 확인해주세요",
       );
-    else setBookId(bookId);
-    closeModal();
+      closeModal();
+    } else setBookId(bookId);
```

Comment:
```suggestion
    } else { 
      setBookId(bookId);
    }
```
괄호를 씌워주면 조금 더 안전해질 것 같아요!